packages/my-joy-beta/src/containers/instances/firewall.js

```diff
@@ -32,13 +32,14 @@ const Firewall = ({
       />
     ));
 
-  const _error = (error && !values.length && !_loading) ? (
-    <Message
-      title="Ooops!"
-      message="An error occurred while loading your instance firewall rules"
-      error
-    />
-  ) : null;
+  const _error =
+    error && !values.length && !_loading ? (
+      <Message
+        title="Ooops!"
+        message="An error occurred while loading your instance firewall rules"
```

Comment:
Use as children pls

Reply:
Ah, thanks. Forgot that the API changed ;)